bot/adapters/rest/rest_message.py

```diff
@@ -0,0 +1,32 @@
+import json
+
+from bot.adapters.rest.models import CommandRequest
+from bot.interfaces.message import AbstractMessage
+
+
+class RestMessage(AbstractMessage):
+    def __init__(self, payload: CommandRequest, user_data: json):
+        payload.json = True
+        self._payload = payload
+        self._user_data = user_data
```

Comment:
to jest protected nie private, przejrzyj to sobie porządnie i uwzględnij różnicę

Reply:
klasy są jak dostęp do danych tajnych w rządzie, dajesz tylko tym którzy potrzebują naprawdę xD